content/docs/command-reference/version.md

```diff
@@ -113,26 +109,23 @@ Inside a DVC project:
 ```dvc
 $ dvc version
 
-DVC version: 0.41.3+f36162
-Python version: 3.7.1
-Platform: Linux-4.15.0-50-generic-x86_64-with-debian-buster-sid
-Binary: False
-Cache: reflink - False, hardlink - True, symlink - True
-Supported remotes: azure, gdrive, gs, hdfs, http, https, s3, ssh, oss
-Filesystem type (cache directory): ('ext4', '/dev/sdb3')
+DVC version: 1.7.4 (pip)
+---------------------------------
+Platform: Python 3.8.3 on Linux-5.4.0-47-generic-x86_64-with-debian-buster-sid
+Supports: All remotes
```

Comment:
> Supports: All remotes

That's still outdated. Did you actually run the command?

Reply:
Yes I checked command on both linux and windows.
This is shown because I installed dvc with: `pip install "dvc[all]"`.

Reply:
For example if I installed DVC using `pip install "dvc[s3]"`, this will be the output:
`Supports: gs, hdfs, http, https, s3`

Reply:
> This is shown because I installed dvc with: pip install "dvc[all]".

I see. I thought it would always list the remotes even if it's all of them. That would be better... But if this is how it works, OK. Let's use an example other than "All remotes" though 🙂 — more interesting.

Reply:
Changed this example. It shows only some remotes now.

Reply:
This is unchanged.

Reply:
Actually I changed this in the next example (on line 130). I added a block quote too which explained the reason. Sorry for confusion.

Reply:
FTFY
```suggestion
Supports: gs, hdfs, http, https, s3
```